### ISSUE TITLE
Migrate to OpenAI API?

### ISSUE THREAD

**Issue by wizd** [epolewski/EricLLM#4]
Greate project! I wonder if it is possible to make the API compatible to OpenAI?

**Comment by epolewski** [comment]
Yeah, you're probably right. The only reason I even went the vLLM format was so that I could easily switch all my stuff back when they or I figured out the GPU spanning bug that's affecting my system. But now people are actually using this so it could go to that OpenAI format.

**Comment by chardog** [comment]
While loading mixtral I get
"AssertionError: Insufficient space in device allocation".

Command I used 
"python ericLLM.py --model ./models/mistralai_Mixtral-8x7B-Instruct-v0.1 --gpu_split 24,24,24,24,24 --max_prompts 8 --num_workers 1 --gpu_balance"

It also doesn't support quantization.
